tox.ini

```diff
@@ -19,7 +19,7 @@ passenv =
     HOME
     PYTHONPATH
 deps =
-    cairo-lang==0.10.0
+    cairo-lang==0.10.1
```

Comment:
I just created https://github.com/OpenZeppelin/cairo-contracts/issues/497. are we 100% sure there's no problem in bumping it with no updates? i think it might be ok

Reply:
I think it shouldn't be any problems reviewing the changelog. And also, I would expect a minor change not to break any integration if the release is correct :).